tests/Functional/Admin/PageAdminTest.php

```diff
@@ -171,9 +171,19 @@ public static function provideFormUrlsCases(): iterable
         yield 'Remove Snapshot Page' => ['/admin/tests/app/sonatapagesnapshot/1/delete', [], 'btn_delete'];
 
         // Block child pages
-        yield 'Create Block Page' => ['/admin/tests/app/sonatapagepage/1/sonatapageblock/create', [
-            'type' => 'sonata.block.service.text',
-        ], 'btn_create_and_list', []];
+        yield 'Create Block Page - Children Pages' => ['/admin/tests/app/sonatapagepage/1/sonatapageblock/create', [
```

Comment:
Here we will only add block creation that can be only tested this way, so we don't repeat too many test for the same thing.